test/comparison-tests/create-and-execute-test.js

```diff
@@ -371,6 +371,8 @@ function getNormalisedFileContent(file, location, test) {
                 .replace(/[\d]+[.][\d]* kB/g, ' A-NUMBER-OF kB')
                 // We also don't want a difference in the number of bytes to fail the build
                 .replace(/ \d+ bytes /g, ' A-NUMBER-OF bytes ')
+                // Sometimes "[built]" is written to output, and sometimes not. This should not fail the build
```

Comment:
Hey!  The first thing to say is that the integration tests are flaky.  As it's a full integration test you do sometimes get false failures.  This happens during the nightly builds.  I've never looked into it in detail but generally rerunning a failing test pack will result in a green run.  Anyway, `[built]` is significant AFAIK as it indicates whether a module has been recompiled as part of a run.

So I'd take the occasional failure here as a "something to live with".